api/ooniapi/probe_services.py

```diff
@@ -661,10 +667,6 @@ def serve_tor_targets() -> Response:
     return nocachejson(torconf)
 
 
-# Unneded: we use an external test helper
-# @probe_services_blueprint.route("/api/private/v1/wcth")
-
-
```

Comment:
You should remove from this patch anything that's touching the legacy API, since it's a bit scary to touch that code. We should keep the changes to that down to the bare minimum.